### PR TITLE
Wizard Sounds has the update. Been a bit since i've done a commit, so…

### PR DESCRIPTION
**Pull request by Cuboos**
Been a while since i've done a pull request, so forgive me if i did this wrong.

Anyway, added a whole bunch of magic sounds to the wizard code, as if wizard wasn't fun enough, now all those whooshes and zaps and lightning bolts. Invocations will come later once we get decent voices. I also added the new title theme, there was a lot of disagreement about how it should sound, so i'm leaving it for now, if it needs to be changed i'll do it later. 

Previews of the sounds. 
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Blind.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/blink.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/CastSummon.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Charge.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/CowHead_Curse.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/HorseHead_curse.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/PigHead_curse.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Disable_Tech.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Disintegrate.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Ethereal_Enter.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Ethereal_Exit.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Fireball.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/FleshToStone.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/ForceWall.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Knock.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/lightning_chargeup.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/lightningbolt.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/LightningShock.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/MAGIC_MISSILE.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/MandSwap.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/MM_Hit.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Mutate.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Repulse.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Smoke.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Staff_animation.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Staff_Change.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Staff_Chaos.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Staff_Door.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Staff_Healing.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Summon_guns.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Summon_Karp.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Summon_Magic.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/SummonItems_generic.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Teleport_app.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Teleport_diss.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/Wand_Teleport.ogg
https://dl.dropboxusercontent.com/u/9654207/Wizard%20Spells/WandODeath.ogg

### CODE REVIEW

**Comment by Iamgoofball** [comment]
no prob(1) to play that audio clip of "MAGIC MISSILE MAGIC MISSILE MAGIC MISSILE" on usage

:-1:

( :+1: )

**Comment by Incoming5643** [comment]
Robeless spells probably shouldn't have sounds attached, it kind of makes using them for their intended use (not going insanely loud) worthless.

**Comment by Iamgoofball** [comment]
make 'em play sounds only if you're wearing robes then

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -47,7 +47,7 @@ var/datum/subsystem/ticker/ticker
 /datum/subsystem/ticker/New()
 	NEW_SS_GLOBAL(ticker)
 
-	login_music = pickweight(list('sound/ambience/title2.ogg' = 49, 'sound/ambience/title1.ogg' = 49, 'sound/ambience/clown.ogg' = 2)) // choose title music!
+	login_music = pickweight(list('sound/ambience/title2.ogg' = 49, 'sound/ambience/title1.ogg' = 49, 'sound/ambience/title3.ogg' =49, 'sound/ambience/clown.ogg' = 2)) // choose title music!
```
Comment (Incoming5643) [inline code/controllers/subsystem/ticker.dm]: 32/32/32/4

Clown must live 2015

**Comment by Cuboos** [comment]
Fire Ball is loud, it's a fire ball, it makes explosions. 
Knock can only be heard by the caster, no one else. 
Force wall isn't very loud + it's still a dead give away that there is a wiz. 
Smoke should pretty quiet. 
Blind, i don't know much about blind, no takes it anyway, might not matter. 
Mind Swap can only be heard by the caster and the victim, no one else. 
Horsehead curse was already a HEY GUYS LOOK AT ME I'M A WIZARD, doesn't mater if it has a noise. 
Charge is only in a spawnable One Use Spellbook, it isn't a usable option. 
Summon Creature swarm just uses the generic summon item sound that Summon Item uses, and that's only when the creatures appear, not when the spell is casted. 
Artificer, same with Summon Item, generic summon item sound.

**Comment by Incoming5643** [comment]
Alright as long as you took stock of that, Knock and mindswap were the big ones in terms of needing to keep absolute hard tells away.

**Comment by PKPenguin321** [comment]
Dude, this is radical.

**Comment by xxalpha** [comment]
:+1:

**Comment by WJohn** [comment]
This is incredible.

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (Cheridan) [inline code/controllers/subsystem/ticker.dm]: Yeah, the point of those numbers is to add up to 100 :P

Comment (Cuboos) [inline code/controllers/subsystem/ticker.dm]: Oh, i didn't realize, i'll fix it real quick

Comment (Iamgoofball) [inline code/controllers/subsystem/ticker.dm]: just 25% prob all of them imo

**Comment by Xhuis** [comment]
hype

**Comment by daman997** [comment]
this is 100% swag :+1:

**Comment by Lumipharon** [comment]
Charge is also used by one of the apprentices (the healing one I think).

**Pull request merged**